Form/Handler/AuthorizeFormHandler.php

```diff
@@ -40,8 +40,8 @@ class AuthorizeFormHandler
     private $requestStack;
 
     /**
-     * @param FormInterface             $form
-     * @param Request|RequestStack|null $requestStack
+     * @param FormInterface                   $form
+     * @param mixed|Request|RequestStack|null $requestStack
```

Comment:
This was added since it's possible to pass anything here. If we limit the types in the doc, the check in the body of the method doesn't make sense since the types were said to already be from that set, why would you check it?